### ISSUE TITLE
Overwriting non existing parameter values 

### ISSUE THREAD

**Issue by redfrexx** [pledge4future/co2calculator#138]
The following tests are run with parameters combinations that are not present in the emission factors. Instead the provided parameters are overwritten with default values (e.g. [`size=average` is used when `size=small` is not there for the last one](https://github.com/pledge4future/co2calculator/blob/0ddea619391e353faf15536c96f2c453d8f92a24/co2calculator/calculate.py#L151-L153) ). I think in the future it would be better to return an error indicating that this parameter does not exist or have a method that tells the users which options are valid (see #137). What do you think?

```
test_calculate.py:131 (test_calc_co2_bus[size: 'small', fuel_type: `diesel`, vehicle_range: 'long-distance'])
distance = 10, size = 'small', fuel_type = 'diesel', occupancy = None
vehicle_range = 'long-distance', expected_emissions = 0.39

test_calculate.py:131 (test_calc_co2_bus[fuel_type: `cng` and size])
distance = 10, size = 'medium', fuel_type = 'cng', occupancy = None
vehicle_range = 'long-distance', expected_emissions = 0.62

test_calculate.py:131 (test_calc_co2_bus[fuel_type: `hydrogen` and size])
distance = 10, size = 'small', fuel_type = 'hydrogen', occupancy = None
vehicle_range = 'local', expected_emissions = 0.25
```

**Comment by veitu** [comment]
Yes, that's a good idea. In the example, I would still do the calculation with size=average, but give a message to the user saying that average is used because size=small does not exist and to give a list of parameters that do exist

**Comment by redfrexx** [comment]
I think I now gives an error message. Check if this is the case.
